src/plugins/graph/components/graph.tsx

```diff
@@ -126,7 +146,6 @@ export const Graph = observer(function Graph({graphController, graphRef, dotsRef
       return <GraphAxis key={place}
                         place={place}
                         enableAnimation={enableAnimation}
-                        autoAdjust={autoAdjustAxes}
```

Comment:
This is how the previous code to rescale axes on data change was broken. I think the new code is more elegant than the old, however, so we can worry about cleaning it up later.